pkg/cmdutil/shell/shell.go

```diff
@@ -0,0 +1,85 @@
+package shell
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/phuslu/log"
+)
+
+func MoveFile(source string, destination string) error {
```

Comment:
I would also propose to move MoveFile and RunCommand functions right to file where they are used, since they are rather small and used only in one place. I also don't think have such util packages is a good idea, since they can bloat with random stuff over time (we have this problem in developer-platform functional tests).
But I don't insist here